examples/simple-asset-transfer/quorum/platform/docker-compose.yml

```diff
@@ -54,6 +54,12 @@ x-quorum-def:
         --nodiscover \
         --verbosity 5 \
         --networkid $${NETWORK_ID} \
+        --rpc \
+        --rpcaddr=0.0.0.0 \
+        --rpcvhosts=node1 \
```

Comment:
I use "*" here, same as in cors. Will it work with `node1` value?

Reply:
Thank you!
`node1`: docker-compose guarantees that the host will resolve to the container's IP based on what you name the container in the compose .yml file (also tested it so I know for a fact that it works).
Could've put `*` but I figured if I can make it work with tighter controls then might as well use the least privilege best practice.